### ISSUE TITLE
RBNF rule parsing error

### ISSUE THREAD

**Issue by rotsee** [papandreou/node-cldr#75]
I think something may be wrong in the RBNF rule parsing, but I don't understand exactly whats going on, so let me just give an example:

```
> cldr.extractRbnfFunctionByType("sv").renderSpelloutCardinalReale(-3)
'minus tre'
> cldr.extractRbnfFunctionByType("sv").renderSpelloutCardinalNeuter(-3)
undefined
```

...but in the data the “minus” pattern is available for neuter as well, by [falling back](https://github.com/papandreou/node-cldr/blob/master/3rdparty/cldr/common/rbnf/sv.xml#L99) [`spellout-numbering`](https://github.com/papandreou/node-cldr/blob/master/3rdparty/cldr/common/rbnf/sv.xml#L25), unless I completely misunderstand something.

**Comment by papandreou** [comment]
Hmm, weird that there is no `-x` rule: https://github.com/papandreou/node-cldr/blob/5f1387dfdc82037a979e1e2e28f1271533f9681e/3rdparty/cldr/common/rbnf/sv.xml#L99-L101

This discussion looks related: http://unicode.org/pipermail/cldr-users/2016-December/000582.html

**Comment by rotsee** [comment]
Oh, I assumed that meant that the minus rule should be inherited too, but now that I read the thread you are linking to I realize that's not the case. I'm closing this to suggest a change at the source instead.

**Comment by papandreou** [comment]
Thanks for seeing this through! If it's causing trouble for you, we can easily add a temporary fix to the checked in copy of cldr.

**Comment by rotsee** [comment]
I'm reopening this as according to the Unicode people this rule should actually be interpreted as inheriting the negative pattern as well, see http://st.unicode.org/cldr-apps/numbers.jsp?locale=sv

**Comment by papandreou** [comment]
Thanks for the exploration. I still don't understand where that `-x` rule is supposed to be inherited from, there's no `spellout-cardinal-neuter` in `root.xml`. The whole thing feels quite underspecified and tied into the ICU implementation.

I played around with that Number Format Tester that you linked to, and it seems like it just uses the `0` rule in the absence of a `-x` rule, so I implemented that and released 5.0.1. Let me know if that works :)
